kahypar/partition/configuration.h

```diff
@@ -46,6 +46,14 @@ struct PreprocessingParameters {
   bool remove_always_cut_hes = false;
   bool remove_parallel_hes = false;
   MinHashSparsifierParameters min_hash_sparsifier = MinHashSparsifierParameters();
+  
+  //Louvain-Configuration
+  bool use_louvain = false;
+  bool use_louvain_in_ip = false;
+  LouvainEdgeWeight louvain_edge_weight = LouvainEdgeWeight::hybrid;
+  bool louvain_use_bipartite_graph = true;
+  int max_louvain_pass_iterations = 10;
+  long double min_eps_improvement = 0.01;
```

Comment:
These parameters should be extracted into a struct and named appropriately.
I'd suggest something like

struct Louvain {
enabled
enabled_in_initial_partitioning
edge_weight
use_bipartite_graph
max_pass_iterations
min_eps_improvement
}